packages/sanity/src/structure/panes/document/statusBar/DocumentStatusLine.tsx

```diff
@@ -76,6 +79,7 @@ export function DocumentStatusLine({singleLine}: DocumentStatusLineProps) {
           published={editState?.published}
           version={editState?.version}
           singleLine={singleLine}
+          currentGlobalBundle={currentGlobalBundle}
```

Comment:
The reason for adding this here is because the `usePaneRouter()` is exported from structure (where this component is). All children components used within are core components and we wouldn't want to do import these hooks within (plus it aids in simplifying components and what they need to care about)